MediaManager.Abstractions/Enums/MediaTrackType.cs

```diff
@@ -0,0 +1,28 @@
+namespace Plugin.MediaManager.Abstractions.Enums
+{
+    public enum MediaTrackType
```

Comment:
What is this type supposed to do? There is already an type to say what video or audio it is.

Reply:
Are you referring to the existing MediaPlaybackType enum of Video and Audio?  Anyway the new MediaTrackType is to abstract these Android and iOS enum for the purpose of selecting audio channel of a video:
https://developer.xamarin.com/api/type/Android.Media.MediaTrackType/
https://developer.xamarin.com/api/type/MonoTouch.AVFoundation.AVMediaType/